integration_tests/integration/add-note.spec.js

```diff
@@ -0,0 +1,98 @@
+import { getEmptyRecallResponse, getRecallResponse } from '../mockApis/mockResponses'
+
+context('Add a note to a recall', () => {
+  const nomsNumber = 'A1234AA'
+  const recallId = '123'
+
+  beforeEach(() => {
+    cy.login()
+  })
+
+  it('can add a note without a document', () => {
+    // TODO: PUD-1489: form handler not yet supporting missing doc.
+  })
+
+  it('can add a note with a document', () => {
+    cy.task('expectGetRecall', {
+      expectedResult: {
+        ...getEmptyRecallResponse,
+        recallId,
+        status: 'AWAITING_DOSSIER_CREATION',
+      },
+    })
+    const note = getRecallResponse.notes[0]
+    cy.task('expectAddNote')
+    cy.visitRecallPage({ nomsNumber, recallId, pageSuffix: 'assess' })
+    cy.clickButton('Actions')
+    cy.clickLink('Add a note')
+    cy.pageHeading().should('equal', 'Add a note to the recall')
+    cy.fillInput('Subject', note.subject)
+    cy.fillInput('Details', note.details)
+    cy.uploadDocx({ field: 'fileName' })
+
+    // lorem-ipsum-msword.docx
+    const generatedDocumentId = '789'
+
+    cy.task('expectGetRecall', {
+      expectedResult: {
+        ...getRecallResponse,
+        recallId,
+        notes: [
+          {
+            noteId: note.noteId,
+            subject: note.subject,
+            details: note.details,
+            index: 1,
+            documentId: generatedDocumentId,
+            fileName: 'lorem-ipsum-msword.docx',
+            createdByUserName: 'Arnold Caseworker',
+            createdDateTime: '2020-12-05T18:33:57.000Z',
+          },
+        ],
+      },
+    })
+    cy.clickButton('Add note')
+    cy.assertSaveToRecallsApi({
+      url: `/recalls/${recallId}/notes`,
+      method: 'POST',
+      bodyValues: {
+        subject: note.subject,
+        details: note.details,
+        fileName: 'lorem-ipsum-msword.docx',
+        // file content was also sent
```

Comment:
I added support for wildcard content; try adding `fileContent: "*"` to see if it works, it will at least check some content is being sent (can always add this later though)